pkg/virt-handler/container-disk/mount.go

```diff
@@ -59,7 +55,6 @@ type Mounter interface {
 	ContainerDisksReady(vmi *v1.VirtualMachineInstance, notInitializedSince time.Time) (bool, error)
 	MountAndVerify(vmi *v1.VirtualMachineInstance) (map[string]*containerdisk.DiskInfo, error)
 	Unmount(vmi *v1.VirtualMachineInstance) error
-	ComputeChecksums(vmi *v1.VirtualMachineInstance) (*DiskChecksums, error)
```

Comment:
During upgrade we will have a situation where old and new handlers will co-live. Therefore here we need to keep producing the checksums. 
@Barakmor1 @acardace

Reply:
Can you please elaborate why?

Reply:
In large clusters the upgrade can take some time. Now VMs running on a new handler will be unable to migrate to old ones that expect the checksum to be computed.

Reply:
Thanks for the note. I'll create a PR to include the checksum calculation for the next two releases, and then we can remove it entirely.

Reply:
https://github.com/kubevirt/kubevirt/pull/14021 
Here is the PR to restore just the checksum calculation without the verification